.github/workflows/build_wamr_lldb.yml

```diff
@@ -93,18 +107,53 @@ jobs:
           cmake --build build --target lldb install --parallel $(nproc)
         working-directory: core/deps/llvm-project
 
+      - name: build lldb macos
+        if: steps.lldb_build_cache.outputs.cache-hit != 'true' && contains(inputs.runner, 'macos')
+        run: |
+          echo "start to build lldb..."
+          mkdir -p inst
+          cmake -S ./llvm -B build \
+            -G Ninja \
+            -DCMAKE_INSTALL_PREFIX=../inst \
+            -DCMAKE_BUILD_TYPE:STRING="Release" \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+            -DLLVM_ENABLE_PROJECTS="clang;lldb" \
+            -DLLVM_INCLUDE_TESTS:BOOL=OFF \
+            -DLLVM_INCLUDE_EXAMPLES:BOOL=OFF  \
+            -DLLVM_BUILD_BENCHMARKS:BOOL=OFF \
+            -DLLVM_BUILD_DOCS:BOOL=OFF \
+            -DLLVM_BUILD_LLVM_DYLIB:BOOL=OFF \
+            -DLLVM_ENABLE_BINDINGS:BOOL=OFF \
+            -DLLVM_TARGETS_TO_BUILD:STRING="X86;WebAssembly" \
+            -DLLVM_ENABLE_LIBXML2:BOOL=ON \
+            -DLLDB_ENABLE_PYTHON:BOOL=OFF \
+            -DLLDB_BUILD_FRAMEWORK:BOOL=OFF
+          cmake --build build --target lldb install --parallel $(nproc)
+        working-directory: core/deps/llvm-project
+
       - name: pack a distribution
         if: steps.lldb_build_cache.outputs.cache-hit != 'true'
         run: |
           mkdir -p inst/bin
           mkdir -p inst/lib
           cp build/bin/lldb* inst/bin
-          cp build/lib/liblldb*.so inst/lib
-          cp build/lib/liblldb*.so.* inst/lib
           cp lldb/tools/lldb-vscode/package.json inst
           cp -r lldb/tools/lldb-vscode/syntaxes/ inst
         working-directory: core/deps/llvm-project
 
+      - name: pack ubuntu specific libraries
```

Comment:
how about  `cp build/lib/liblldb*.* inst/lib`, and merge below two steps?

Reply:
I've made the macOS copy a little stricter since first revision as there are a bunch of static libs added to this folder, which don't need to be included https://pastebin.com/tpbjTXyt

Reply:
Got your point.